Zifan94/TeamWHOAMI
# add plsClose and change the order of M4 M5 M6

WHAT I DID:
1. Add PlsClose packet sent and receive.
2. Change the order of M4, M5, M6
 original: C -- M1 --> S
                C <-- M2 -- S
                C -- M3 --> S
                C <-- M4 -- S
                C <-- M5 -- S
                C -- M6 --> S  

 now      : C -- M1 --> S
                C <-- M2 -- S
                C -- M3 --> S
                C <-- M4 -- S
                C -- M5 --> S
                C <-- M6 -- S       

Test Plan:
1. Run main program successfully
2. Change some line in PlsClientProtocol or PlsServerProtocol and make it into error state and see the log
